arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2394,6 +2394,23 @@ Result ClusterInfo::checkCollectionPreconditions(std::string const& databaseName
   return {};
 }
 
+Result ClusterInfo::incrementPlanVersion() {
+  auto trx = AgencyWriteTransaction{
+    std::vector<AgencyOperation>{IncreaseVersion()}, std::vector<AgencyPrecondition>{}};
+  AgencyComm ac(_server);
+  auto res = ac.sendTransactionWithFailover(trx);
+  
+  LOG_TOPIC("ccb89", DEBUG, Logger::CLUSTER) << "incrementing plan version";
+
+  if (res.successful()) {
+    if (res.slice().get("results").length() > 0) {
+      Result r = waitForPlan(res.slice().get("results")[0].getNumber<uint64_t>()).get();
+      return r.fail() ? r : Result{};
```

Comment:
`waitForPlan.get()` already returns a result, so we can return it as it is.
```suggestion
      return waitForPlan(res.slice().get("results")[0].getNumber<uint64_t>()).get();
```